docs/cloud/google-cloud.rst

```diff
@@ -0,0 +1,52 @@
+Running Mango on Google Cloud
+=============================
+
+`Cloud Dataproc <https://cloud.google.com/dataproc/>`__ sets up the basic environment with HDFS and Spark, providing a simple environment to run Mango.
+
+Commands in this section will require users to create an account on `Google Cloud <https://cloud.google.com/>`__ and  install the `gcloud cli <https://cloud.google.com/sdk/gcloud/>`__
+
+Creating a Dataproc Cluster
+---------------------------
+Download the necessary initialization scripts:
+
+.. code:: bash
+
+    wget https://gist.githubusercontent.com/Georgehe4/6bb1c142a9f68f30f38d80cd9407120a/raw/9b903e3b8746ee8f25911fe98925b53e9777002f/mango_install.sh
```

Comment:
Can we put this script in Mango scripts?

Reply:
Added